mantidproject/mantid
# Indirect - Reduction - ASCII Saving - Save ASCII in 7 column format

Confusion surrounding requests from the TOSCA instrument scientists led to changing the ASCII saving version to version 2 in IndirectReductionCommon - 3 column format.

This change was asked to be reverted by the TOSCA instrument scientists - and so has been changed back to version 1.

**To test:**
1. Navigate to Interfaces > Indirect > Data Reduction.
2. Ensure the selected instrument is TOSCA.
3. Enter 17411 into the Run Files field.
4. Check the checkbox labelled 'ASCII' found under Select Save Formats.
5. Run the algorithm and then click save in the interface.
6. Open the saved file in your default save directory and check it is saved in 7 column format.

Fixes #20098. 

**Release Notes** 

Added change to the release notes.
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
